src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createDriverRequest =
+                "INSERT INTO taxi_db.drivers (name, licenseNumber) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement =
+                         connection.prepareStatement(
+                             createDriverRequest, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenseNumber());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can not create driver " + driver,e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getDriverRequest =
+                "SELECT * FROM taxi_db.drivers WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getDriverRequest)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            Driver driver = null;
+            if (resultSet.next()) {
+                driver = getDriver(resultSet);
+            }
+            return Optional.ofNullable(driver);
+        } catch (SQLException e) {
+            throw new RuntimeException("Can not get driver with id - " + id, e);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String getAllDriversRequest =
+                "SELECT * " + "FROM taxi_db.drivers" + "WHERE is_deleted = false";
```

Comment:
```suggestion
        String getAllDriversRequest =
                "SELECT * FROM taxi_db.drivers WHERE is_deleted = false";
```

fix in all places

Reply:
I understand, for me easier to see like this
SELECT*
FROM ....
WHERE

that's why I used +